RecoTauTag/RecoTau/interface/AntiElectronIDMVA6.h

```diff
@@ -32,98 +33,82 @@
 
 #include <vector>
 
+struct TauVars {
```

Comment:
The names are a bit simple; consider adding a namespace or at least check that there are no possible name collisions